news/8733.bugfix

```diff
@@ -0,0 +1 @@
+Correctly uninstall console scripts installed to ~/.local/bin
```

Comment:
This should be modified to avoid saying both *console scripts* and `~/.local/bin`. The issue affects more than console scripts, and `~/.local/bin` does not apply to Windows. I would suggest replacing them with *scripts* and *installed with `--user`*.